key,default_value
userAgent,"Boto3/1.4.7 Python/2.7.13 Darwin/16.7.0 Botocore/1.7.38"
eventID,"e06ee5d5-ab5d-4ae2-a580-64cdc341e3cd"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: organizations:DescribeAccount on resource: arn:aws:organizations::745598359964:account/o-57vb65iy55/811596193553"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"1.198.254.60"
eventName,"DescribeAccount"
eventSource,"organizations.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"5dbbdf94-c1be-11e7-9b01-67c4270b379a"
responseElements,"None"
eventVersion,"1.04"
eventTime,"2017-11-05T00:15:01Z"
